.config/CredScanSuppressions.json

```diff
@@ -53,7 +53,11 @@
                 "src/libraries/System.Net.Http/tests/UnitTests/HttpEnvironmentProxyTest.cs",
                 "src/libraries/System.Net.Mail/tests/Functional/SmtpClientTest.cs",
                 "src/libraries/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs",
-                "src/libraries/System.Security.Cryptography.Xml/tests/TestHelpers.cs"
+                "src/libraries/System.Security.Cryptography.Xml/tests/TestHelpers.cs",
+                "src/libraries/System.Security.Cryptography.X509Certificates/tests/PfxTests.cs",
+                "src/libraries/System.Management/src/System/Management/ManagementScope.cs",
```

Comment:
This file, according to the path, doesn't look like a test code. Is there a reason we cannot do inline suppression for it instead?